app/src/main/java/org/mozilla/reference/browser/ext/Context.kt

```diff
@@ -23,3 +28,21 @@ val Context.components: Components
 
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
+
+/**
+ *  Creates a share intent returns true it is able to share false otherwise.
+ */
+fun Context.share(text: String, subject: String = ""): Boolean {
+    return try {
+        val intent = Intent(Intent.ACTION_SEND)
+        intent.type = "text/plain"
+        intent.putExtra(EXTRA_SUBJECT, subject)
+        intent.putExtra(EXTRA_TEXT, text)
+        intent.flags = FLAG_ACTIVITY_NEW_TASK
+        startActivity(intent)
+        true
+    } catch (e: ActivityNotFoundException) {
+        e.printStackTrace()
```

Comment:
We already have our Log component set up in the reference browser. Would be good to use it here: `mozilla.components.support.base.log.Log`

Reply:
👍

Reply:
I borrow the implementation from [Anko Project](https://github.com/Kotlin/anko/). Like this there're many cool functionality that they provide. I'm not sure if we should include it on A-C or should we suggest apps that use directly [Anko Project](https://github.com/Kotlin/anko/). This way this is less code to maintain as Anko is wide use for Android Dev